src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      calculator.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      calculator.result -= number;
+
+      return this;
+    },
+    multiply(number) {
+      calculator.result *= number;
+
+      return this;
+    },
+    divide(number) {
+      calculator.result /= number;
+
+      return this;
+    },
+    operate(operation, number) {
+      operation(number);
```

Comment:
The `operate` method directly calls the `operation` function with `number` as an argument. It would be safer to check if `operation` is a valid function before calling it to prevent runtime errors.